File: src/kyu_8/swap_values/js/swapValuesTest.js
const Test = require('@codewars/test-compat');
const swapValues = require("./swapValues");

describe("Tests", () => {
    it("test", () => {
        let arr = [1, 2]
        swapValues(arr);
        Test.assertEquals(arr[0], 2, "Failed swapping numbers")
        Test.assertEquals(arr[1], 1, "Failed swapping numbers")

    });
});

File: src/kyu_8/swap_values/csharp/swap_values_test.cs
namespace Solution
{
    using NUnit.Framework;

    using System;
    using System.Linq;

    [TestFixture]
    public class Tests
    {
        [Test]
        public void ExampleTest()
        {
            int[] args = new[] { 1, 2 };

            Swapper swapper = new Swapper(args.Cast<object>().ToArray());
            swapper.SwapValues();

            Assert.AreEqual(2, swapper.Arguments[0], "Failed swapping numbers");
            Assert.AreEqual(1, swapper.Arguments[1], "Failed swapping numbers");
        }
    }
}